src/js/schemas/service-schema/ContainerSettings.js

```diff
@@ -4,15 +4,19 @@ import React from 'react';
 
 const ContainerSettings = {
   title: 'Container Settings',
-  description: 'Configure your Docker Container. You can configure your Docker Volumes in the Volumes tab and your Docker Ports in the Network tab.',
+  description: 'Configure your Docker Container. You can configure your Docker volumes in the Volumes tab and your Docker ports in the Network tab.',
   type: 'object',
   properties: {
     basic: {
       type: 'group',
       properties: {
         image: {
-          description: 'name of your docker image',
-          title: 'Image',
+          description: ()
+            <span>
+            Configure your Docker container. Use <a href="https://hub.docker.com/explore/">DockerHub</a> to find popular repositories.
```

Comment:
Let's change the link to `<a href="https://hub.docker.com/explore/" target="_blank>" so that the link will open in a new tab. Also this specific line (#16) should be indented once further.